fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -1906,6 +1906,160 @@ countries:
                 issue_months: [0, 1, 2, 3, 4, 5]
                 start_year: 1991
 
+    mali:
+        logo: Mali_IRI_120x48.png
```

Comment:
Do you have a logo file? I have to add that to the server (separately, through a different mechanism).